tracktotrip/location.py

```diff
@@ -216,7 +216,7 @@ def infer_location(
         queried_locations = location_query(point, max_distance)
         for (label, centroid, _) in queried_locations:
             locations.append({
-                'label': unicode(label, 'utf-8'),
+                'label': str(label, 'utf-8'),
```

Comment:
A label might have a unicode character

Reply:
Literal strings are unicode by default in Python3. Unicode of Python2 is equivalent to str in Python3

Reply:
yes, but that will break compatibility with python 2.x